pypeerassets/card_parsers.py

```diff
@@ -1,15 +1,27 @@
 '''parse cards according to deck issue mode'''
 
-from .pautils import exponent_to_amount
+from typing import Callable, Optional
 
 
-def none_parser(cards):
+def amount_to_exponent(amount: float, number_of_decimals: int) -> int:
+    '''encode amount integer as exponent'''
+
+    return int(amount * 10**number_of_decimals)
+
+
+def exponent_to_amount(exponent: int, number_of_decimals: int) -> float:
```

Comment:
This belongs in pautils, as it's used by some other tooling from there too. It's a utility function.